src/util/util.ts

```diff
@@ -47,3 +48,44 @@ export const getMedicationSpecificRemsAdminUrl = (
 
   return cdsUrl.remsAdmin;
 };
+
+const getIntermediaryCdsUrl = (hook: SupportedHooks): string => {
+  const intermediaryBaseUrl = env.get('INTERMEDIARY_SERVER_BASE').asString();
+  if (!intermediaryBaseUrl) {
+    throw new Error(
+      'Forwarding CDS hook to REMS intermediary, but INTERMEDIARY_SERVER_BASE is not defined'
+    );
+  }
+  return `${intermediaryBaseUrl}/r4/cds-services/${hook}-crd`;
```

Comment:
This is the wrong path for the rems hooks. It should be something like:

`return `${intermediaryBaseUrl}/cds-services/rems-${hook}`;`

Reply:
Just updated the path in commit https://github.com/mcode/rems-smart-on-fhir/pull/105/commits/a76f16ec68beb563823bd114b8fcfc0314e70413